src/vision.cs

```diff
@@ -2265,11 +2379,22 @@ interface VNClassifyImageRequest {
 		[DesignatedInitializer]
 		IntPtr Constructor ([NullAllowed] VNRequestCompletionHandler completionHandler);
 
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use 'GetSupportedIdentifiersAndReturnError' instead.")]
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Use 'GetSupportedIdentifiersAndReturnError' instead.")]
+		[Deprecated (PlatformName.TvOS, 15, 0, message: "Use 'GetSupportedIdentifiersAndReturnError' instead.")]
 		[Static]
 		[Export ("knownClassificationsForRevision:error:")]
 		[return: NullAllowed]
 		VNClassificationObservation [] GetKnownClassifications (VNClassifyImageRequestRevision revision, [NullAllowed] out NSError error);
 
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("supportedIdentifiersAndReturnError:")]
+		[return: NullAllowed]
+		string[] GetSupportedIdentifiersAndReturnError ([NullAllowed] out NSError error);
```

Comment:
remove `AndReturnError` suffix

Reply:
not sure this should return `string[]`
maybe `NSString[]` since they seems to be constants
or better an `enum` if one exists (or was added)